lib/test.gi

```diff
@@ -371,6 +371,14 @@ InstallGlobalFunction("Test", function(arg)
            end,
            subsWindowsLineBreaks := true,
          );
+
+  if IsBound(HPCGAP) then
```

Comment:
Notes on the first commit message: "Disable the ... method" -> "Disable the ... method for HPC-GAP"

"sometimes add" -> "sometimes adds"